tap_awin/streams.py

```diff
@@ -115,10 +120,6 @@ class TransactionsStream(AwinStream):
         )),
         th.Property("transactionParts", th.ArrayType(
             th.ObjectType(
-                th.Property("advertiserCost", th.ObjectType(
```

Comment:
could this not in theory exist if there's multiple parts to the transaction entry?

Reply:
From what I could tell, it looked like this was removed in the latest version of the API, so even if there were multiple parts to the transaction, the `advertiserCost` would be separated out into the top-level / not included in those `transactionParts`, right?

Reply:
And I could very well be wrong. I'm not against keeping this in there to avoid breaking things for anyone, but I just wasn't able to trace that logic through the [API docs](https://wiki.awin.com/index.php/API_get_transactions_list).